src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortBamSparkIntegrationTest.java

```diff
@@ -1,15 +1,60 @@
 package org.broadinstitute.hellbender.tools.spark.pipelines;
 
+import htsjdk.samtools.ValidationStringency;
 import org.broadinstitute.hellbender.CommandLineProgramTest;
 import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
 import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
 import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.File;
 
 public final class SortBamSparkIntegrationTest extends CommandLineProgramTest {
```

Comment:
yes, but for now the practice is to have spark and non-spark tools have corresponding names.